tests/inc/class-outcomestestcase.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Candela;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class OutcomesTestCase
+ *
+ * @package CandelaOutcomes
+ */
+
+class OutcomesTestCase extends \PHPUnit\Framework\TestCase {
+	/**
+	 * A single example test.
+	 */
+	function test_sample() {
```

Comment:
When would actual tests be added to the project?

Reply:
I can take some time today/tomorrow to _not_ cut corners and actually write a few tests that will go out with this deploy.  :)

Reply:
That would be great.  Everything otherwise looks good and I will rereview after.